### PR TITLE
chore: Update documentation following my journey with getting lok built

### PR DESCRIPTION
**Pull request by whutchinson98**

### CODE REVIEW

**Comment by chase** [comment]
Maybe we should just change the instructions to use only MacPorts :shrug: 

That's what's used in our macOS build servers anyway.

**Comment by whutchinson98** [comment]
> Maybe we should just change the instructions to use only MacPorts 🤷
> 
> That's what's used in our macOS build servers anyway.

Yeah I think that would be a good idea. I'll update the readme accordingly

File: README.md [hunk 1]
```diff
@@ -2,40 +2,60 @@
...
+
+- [Install MacPorts](https://www.macports.org/install.php)
+- Install dependencies using MacPorts:
+
+```shell
+sudo port install gperf
+sudo port install gpatch
+sudo port install nasm
+sudo port install autoconf
+sudo port install automake
```
Comment (chase) [inline README.md:34]: You can do `port install gperf gpatch ...`, multiple packages on one line

Comment (whutchinson98) [reply README.md]: Updated doc

**Pull request merged**